lib/tasks/data_migrations/migrate_contact_metadata_to_native_fields.rb

```diff
@@ -0,0 +1,42 @@
+namespace :data_migrations do
+  task migrate_contact_metadata_to_native_fields: :environment do
+    Account.find_each do |account|
+      puts "Migrating contacts for account: #{account.id}"
+
+      account.contacts.find_each do |contact|
```

Comment:
`account.contacts.where(iso_country_code: nil)` This will allow us to run it multiple times in case of an error

Reply:
Ah, yes. We can say if `iso_country_code` has the value, we already migrated or they created from the new codes.